server/src/assets/game/script-game.js

```diff
@@ -40,20 +44,35 @@ function disableAllDestinationButtons() {
 		$(`#${destinationNamePrefix}${i}`).prop('disabled', true);
 		$(`#${destinationNamePrefix}${i}`).removeClass(destinationEnabledButtonStyle);
 		$(`#${destinationNamePrefix}${i}`).addClass(destinationDisabledButtonStyle);
+		document.getElementById(`${destinationNamePrefix}${i}`).innerHTML = "<p>Keine Route</p>";
 	}
 }
 
+function getImageSrcFlag(destinationSignal){
+	let FlagData = signalToFlagMapping[destinationSignal];
+	if(FlagData != null){
+		console.log(flagRootLocation + FlagData["file"]);
+		return "<img src='" + flagRootLocation + FlagData["file"] + "' style='width: 35px; height: 35px;'/>";
+	}
+	FlagData = signalToFlagMapping[destinationSignal.substring(0, destinationSignal.length-1)];
```

Comment:
Why not use the proper names of the destination signals in signalToFlagMapping?

Reply:
removed